assets/src/settings-page/index.js

```diff
@@ -132,13 +191,48 @@ function Root() {
 				saveOptions();
 			} }>
 				<TemplateModes />
-				<h2>
+				<h2 id="advanced-settings">
 					{ __( 'Advanced Settings', 'amp' ) }
 				</h2>
 				<MobileRedirection />
-				<SupportedTemplates />
-				<PluginSuppression />
-				<Analytics />
+				<AMPDrawer
+
+					heading={ (
+						<h3>
+							{ __( 'Supported Templates', 'amp' ) }
+						</h3>
+					) }
+					hiddenTitle={ __( 'Supported templates', 'amp' ) }
+					id="supported-templates"
+					initialOpen={ 'supported-templates' === focusedSection }
+				>
+					<SupportedTemplates />
```

Comment:
Same question as above.

Reply:
In this case the ID is set just above on the `AMPDrawer` component.